test/api/config-manager.test.js

```diff
@@ -0,0 +1,103 @@
+const { isString, isNumber, isArray, isBoolean } = require('lodash');
+const Chai = require('chai');
+const sinon = require('sinon');
+const ChaiAsPromised = require('chai-as-promised');
+
+const ConfigManager = require('../../src/api/config-manager');
+const { getContract } = require('./mock/contract');
+
+Chai.use(ChaiAsPromised);
+const assert = Chai.assert;
+const expect = Chai.expect;
+
+const eventAddress = '0x8b2b1838efff78e5ed9e00d95d9ef071f2c27be6';
+const address = '0x939592864C0Bd3355B2D54e4fA2203E8343B6d6a';
+
+describe('api/config-manager', () => {
+  let stubGetContract;
+  beforeEach(() => {
+      stubGetContract = sinon.stub(ConfigManager, 'getContract').callsFake(getContract);
+  });
+
+  afterEach(() => {
+    stubGetContract.restore();
+  });
+
+  describe('bodhiTokenAddress()', () => {
+    it('It returns the bodhiTokenAddress', async () => {
+      const res = await ConfigManager.bodhiTokenAddress();
+      assert.isDefined(res);
+      assert.isTrue(isString(res));
+    });
+  });
+
+  describe('eventFactoryAddress()', () => {
+    it('It returns the eventFactoryAddress', async () => {
+      const res = await ConfigManager.eventFactoryAddress();
+      assert.isDefined(res);
+      assert.isTrue(isString(res));
+    });
+  });
+
+  describe('eventEscrowAmount()', () => {
+    it('It returns the eventEscrowAmount', async () => {
+      const res = await ConfigManager.eventEscrowAmount();
+      assert.isDefined(res);
+      assert.isTrue(isString(res));
+      assert.isFalse(isNaN(res));
```

Comment:
nit: this test technically not needed since the isString test will ensure that this will never be true.